docs/quickstart.md

```diff
@@ -71,42 +71,42 @@ For more information on how to write the SubQuery, check out our documentation u
 
 In order to index your SubQuery project, you must first generate the required GraphQL models that you have defined in your GraphQL Schema file (`schema.graphql`). Run this command in the root of the project directory.
 ```shell
-# NPM 
-npm run-script codegen
-
 # Yarn
 yarn codegen
+
+# NPM 
+npm run-script codegen
 ```
 
 You'll find the generated models in the `/src/types/models` directory
 
 ## Build the Project
 
-In order run your SubQuery Project on a SubQuery node or our hosted service, you need to build and pack your work.
+In order run your SubQuery Project on a SubQuery node or our hosted service, you need to build your work.
 
-Run pack command from the project's root directory to automatically generate a `PROJECT_NAME.tgz` file.
+Run the build command from the project's root directory to automatically generate a `PROJECT_NAME.tgz` file.
```

Comment:
build command won't generate .tgz file, pack command will. and pack command is now deprecated.

build command is required to run at least once before run `docker-compose`. but not necessary for our hosted service.